usersync/cookie_test.go

```diff
@@ -338,31 +338,54 @@ func TestGetUIDsWithNilCookie(t *testing.T) {
 }
 
 func TestTrimCookiesClosestExpirationDates(t *testing.T) {
+	var expirationTime = time.Date(2021, time.January, 3, 15, 25, 0, 0, time.UTC)
```

Comment:
Using a random constant time with a constant time zone so that the length of the JSON encoded expiry time string is constant and the output of the cookie trim function is predictable.

Reply:
A static time is a good choice. Why this date though? How about a simpler `time.UnixMilli(1)`?